src/runtime/providers/vercel.ts

```diff
@@ -3,7 +3,18 @@ import { stringifyQuery } from 'ufo'
 
 // https://vercel.com/docs/more/adding-your-framework#images
 
-export const getImage: ProviderGetImage = (src, { modifiers, baseURL = '/_vercel/image' } = {}) => {
+export const getImage: ProviderGetImage = (src, { modifiers, baseURL = '/_vercel/image' } = {}, ctx) => {
+  if (process.env.NODE_ENV === 'development') {
+    if (!modifiers?.width) {
+      console.warn(`A defined width must be provided to use the \`vercel\` provider. Warning originated from \`${src}\`.`)
+      return { url: '/_vercel/width-required' }
```

Comment:
What about preserving original URL to avoid 404 (also in production)

Reply:
Great idea. Implemented snapping to the nearest larger size, which feels like the best DX (+ of course warning in dev mode to add desired screen size to config remains).